### PR TITLE
Feature/57 admin promotion revoke of permissions

### PR DESCRIPTION
**Pull request by Rodrigo-Schar**
- add users list
- add promote and revoke user permission
- merge develop
- add unit test 
- fix navigation

### CODE REVIEW

File: app/src/main/java/com/jalasoft/routesapp/ui/adminPages/userPermissions/view/PromoteUsersFragment.kt [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    }
+
+    private fun setRecycler() {
+        binding.recyclerUsersAdministrator.layoutManager = LinearLayoutManager(requireContext())
+        binding.recyclerUsersAdministrator.adapter = UserAdapter(mutableListOf(), this)
+    }
+
+    override fun addRevokeUserPermission(user: User) {
+        if (user.type == 0) {
+            binding.fadUserPermissions.visibility = View.VISIBLE
```
Comment (jmacboy) [inline app/src/main/java/com/jalasoft/routesapp/ui/adminPages/userPermissions/view/PromoteUsersFragment.kt]: if and else have the same value here, add the first line outside the if

**Review by jmacboy — APPROVED** [review]
just that change and you're good to go

**Review by YawarValeriano — APPROVED** [review]
Looks good

File: app/src/main/java/com/jalasoft/routesapp/ui/adminPages/userPermissions/view/PromoteUsersFragment.kt (hunk 1 above)
Comment (Rodrigo-Schar) [reply app/src/main/java/com/jalasoft/routesapp/ui/adminPages/userPermissions/view/PromoteUsersFragment.kt]: done!

**Pull request merged**